src/app/Caches/Create/FeaturesSelector.tsx

```diff
@@ -9,118 +9,119 @@ import {
-                  isOpen={isOpenCacheFeature}
-                  aria-labelledby="cache-feature"
-                  placeholderText={t('caches.create.configurations.feature.cache-feature-list-placeholder')}
-                  toggleId="featuresSelect"
-                  chipGroupProps={{ numChips: 6 }}
-                >
-                  {cacheFeatureOptions()}
-                </Select>
-              </FormGroup>
-            </FormSection>
-            {displayAlert()}
-            {configuration.feature.cacheFeatureSelected.includes(CacheFeature.BOUNDED) && <BoundedCacheConfigurator />}
-            {configuration.feature.cacheFeatureSelected.includes(CacheFeature.INDEXED) && <IndexedCacheConfigurator />}
-            {configuration.feature.cacheFeatureSelected.includes(CacheFeature.SECURED) && <SecuredCacheConfigurator isEnabled={!notSecured} />}
-            {configuration.feature.cacheFeatureSelected.includes(CacheFeature.BACKUPS) && <BackupsCacheConfigurator isEnabled={isBackups} />}
-            {configuration.feature.cacheFeatureSelected.includes(CacheFeature.TRANSACTIONAL) && <TransactionalCacheConfigurator isEnabled={configuration.basic.mode === CacheMode.SYNC} />}
-            {configuration.feature.cacheFeatureSelected.includes(CacheFeature.PERSISTENCE) && <PersistentCacheConfigurator />}
-        </Form>
+      <FormAlert>
+        <Alert variant="warning"
+          title={t('caches.create.configurations.feature.alert-error')}
+          aria-live="polite"
+          isPlain
+          isInline />
+      </FormAlert>
     );
+  }
+
+  return (
+    <Form isWidthLimited onSubmit={(e) => {
+      e.preventDefault();
+    }}>
+      <FormSection title={t('caches.create.configurations.feature.cache-feature-list', { brandname: brandname })}>
+        <FormGroup fieldId='cache-feature'>
+          <Select
+            variant={SelectVariant.typeaheadMulti}
+            typeAheadAriaLabel={t('caches.create.configurations.feature.cache-feature-list-typeahead')}
+            onToggle={() => setIsOpenCacheFeature(!isOpenCacheFeature)}
+            onSelect={onSelectFeature}
+            onClear={onClearFeatureSelection}
+            selections={configuration.feature.cacheFeatureSelected}
+            isOpen={isOpenCacheFeature}
+            aria-labelledby="cache-feature"
+            placeholderText={t('caches.create.configurations.feature.cache-feature-list-placeholder')}
+            toggleId="featuresSelect"
+            chipGroupProps={{ numChips: 6 }}
+          >
+            {cacheFeatureOptions()}
+          </Select>
+        </FormGroup>
+      </FormSection>
+      {displayAlert()}
+      {configuration.feature.cacheFeatureSelected.includes(CacheFeature.BOUNDED) && <BoundedCacheConfigurator />}
+      {configuration.feature.cacheFeatureSelected.includes(CacheFeature.INDEXED) && <IndexedCacheConfigurator isEnabled={schemas.length > 0 && configuration.basic.encoding !== 'text/plain'} />}
```

Comment:
this is wrong. Indexing types are for protobuf encoding. You don't need to passe this as an enabled.